client/client.go

```diff
@@ -78,10 +64,39 @@ type Client struct {
 	Recipients         Recipients
 }
 
+func DefaultConfig() *Config {
+	c := &Config{
+		APIKey:     os.Getenv("HONEYCOMB_API_KEY"),
+		APIUrl:     DefaultAPIHost,
+		Debug:      false,
+		HTTPClient: cleanhttp.DefaultPooledClient(),
+		UserAgent:  defaultUserAgent,
+	}
+
+	// if API Key is still unset, try using the legacy environment variable
+	if c.APIKey == "" {
+		c.APIKey = os.Getenv("HONEYCOMBIO_API_KEY")
```

Comment:
nit: what would you think of moving these `HONEYCOMB_API_KEY` & `HONEYCOMBIO_API_KEY` strings into some constants? I definitely thought they were the same value for a solid 5 minutes until I noticed the `IO` in the legacy env var 😅

Reply:
Done!